### PR TITLE
Defined colors and variables to specific elements to resolve #13

### LINKED ISSUE #13: Move all colors and variables to _colors.css
so that we can update the color schema of the application!

### PR DESCRIPTION
**Pull request by JoacimNilsen**
Hope this looks like something we talked about!

### CODE REVIEW

File: src/Components/SSInput.css [hunk 1]
```diff
@@ -16,7 +11,7 @@
 .SSInput:focus,
 .SSInput:active,
 .SSInput:hover {
-  border: 2px solid var(--active-border);
+  border: 2px solid var(--active-input-border);
```
Comment (erikpantzar) [inline src/Components/SSInput.css:14]: Really nice! But state is usually put in last position.

`--input-border-active`

**Review by erikpantzar — CHANGES_REQUESTED** [review]
Really nice, please change from comment!

File: src/Components/SSInput.css (hunk 1 above)
Comment (JoacimNilsen) [reply src/Components/SSInput.css]: I understand! Changes made

**Review by erikpantzar — APPROVED** [review]
Niiiice! :D

**Pull request merged**